sdk/postgresql/Azure.ResourceManager.PostgreSql/src/autorest.md

```diff
@@ -256,6 +247,25 @@ rename-mapping:
   ServerBackupListResult: PostgreSqlFlexibleServerBackupListResult
   StorageTierCapability: PostgreSqlFlexibleServerStorageTierCapability
   UserAssignedIdentity: PostgreSqlFlexibleServerUserAssignedIdentity
+  AdminCredentials: PostgreSqlMigrationAdminCredentials
+  BackupSettings: PostgreSqlFlexibleServerBackupSettings
+  BackupStoreDetails: PostgreSqlFlexibleServerBackupStoreDetails
+  CancelEnum: PostgreSqlMigrationCancelEnum
+  CapabilityStatus: PostgreSqlFlexbileServerCapabilityStatus
+  DbServerMetadata: PostgreSqlServerMetadata
+  FastProvisioningSupportedEnum: PostgreSqlFlexibleServerFastProvisioningSupportedEnum
+  FlexibleServerCapability.fastProvisioningSupported: SupportFastProvisioning
+  FlexibleServerCapability: PostgreSqlFlexibleServerCapabilityProperties
+  FlexibleServersLogFile: PostgreSqlFlexibleServersLogFile
+  FlexibleServersLogFileListResult: PostgreSqlFlexibleServersLogFileListResult
+  GeoBackupSupportedEnum: PostgreSqlFlexibleServerGeoBackupSupportedEnum
+  HaMode: PostgreSqlFlexibleServerHAMode
+  ZoneRedundantHaSupportedEnum: PostgreSqlFlexibleServerZoneRedundantHaSupportedEnum
+  ZoneRedundantHaAndGeoBackupSupportedEnum: PostgreSqlFlexibleServerZoneRedundantHaAndGeoBackupSupportedEnum
+  StorageAutoGrowthSupportedEnum: PostgreSqlFlexibleServerStorageAutoGrowthSupportedEnum
+  OnlineResizeSupportedEnum: PostgreSqlFlexibleServerOnlineResizeSupportedEnum
+  RestrictedEnum: PostgreSqlFlexibleServerZoneRedundantRestrictedEnum
```

Comment:
I suppose the `Enum` suffix is not a very good name, shall we just remove the suffix?
For instance, for the use case in CancelEnum, we will have something like:
```csharp
var data = new SomeModel()
{
Cancel = PostgreSqlMigrationCancel.True
}
```
which is also intuitive.

Reply:
Done